### ISSUE TITLE
Generating configurations

### ISSUE THREAD

**Issue by Vladimir7280** [nemtech/symbol-bootstrap#77]
Generating configurations in symbol-bootstrap and for compiling from source are different now. 
Question: Will it be the same in release?

**Comment by fboucquez** [comment]
Hi @Vladimir7280 , thank you for your question. 

Atm, bootstrap doesn't compile the server but you can use it to configure. You can compile server like you do regularly and use the config command from bootstrap to create the properties and keys. Then use the compiled binaries to run it. 

You may need to tune the paths (like `catapultAppFolder`) a little bit with a custom preset but it should be fine. It will depends on where do you want to copy your server libs and binaries. 

What do you mean by different configurations?

**Comment by Vladimir7280** [comment]
Hi @fboucquez. 
Thank you for your attention.

Yes, I compiled and successfully launched the catapult server from the source codes (I did this on both x86_64 and ARM8 (Raspberry Pi 4 (4GB)).
I created configurations and everything works.

BUT, I also tested symbol-bootstrap. In various configurations.

I noticed the following differences:

the compiled server requires in the file **config-node.properties** :

``` 
............
maxCacheDatabaseWriteBatchSize = 5MB
maxTrackedNodes = 5'000

minPartnerNodeVersion =
maxPartnerNodeVersion =

#all hosts are trusted when list is empty
trustedHosts =
localNetworks = 127.0.0.1
listenInterface = 0.0.0.0

[localnode]

..........
```

But, the symbol-bootstrap creates a configuration **config-node.properties**

```
........
maxCacheDatabaseWriteBatchSize = 5MB
maxTrackedNodes = 5'000

# all hosts are trusted when list is empty
trustedHosts = 127.0.0.1, 172.20.0.25
localNetworks = 127.0.0.1, 172.20.0.25

[localnode]
......
```
without parameter
_minPartnerNodeVersion =
maxPartnerNodeVersion =
trustedHosts =_

without these parameters, **catapult.server** will not start. He demands them.

**Comment by fboucquez** [comment]
It seems you have a version mismatch between bootstrap and server. What's your bootstrap version? Are you compiling the server from main or from a tag?

Bootstrap 0.2.x should be compatible with the following server tag:
https://github.com/nemtech/catapult-server/tree/v0.10.0.3

Bootstrap 0.3.x (in dev) will be compatible with v0.10.0.4 currently in development

**Comment by Vladimir7280** [comment]
YES!  
```
Copyright (c) Jaguar0625, gimre, BloodyRookie, Tech Bureau, Corp.
catapult version: 0.10.0.4 5c52fbaf [main]
```
&
```
symbol-bootstrap/0.2.1 linux-x64 node-v12.20.0
```

But I was compiling from main.  Strange

**!!!Thank you very much for your answer!!!**
(I'm still waiting for a new version of SDK, the current one does not support the network type 128 (private test))

**Comment by fboucquez** [comment]
Cool, let us know how PI is going. Keen to add support eventually. 

New networks types are in alpha/dev ready to be released once we finish 10.0.4 support and the feature we want to implement
